pkgs/servers/mail/listmonk/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, buildGoModule, fetchFromGitHub, callPackage, stuffbin }:
+let
+  frontend = callPackage ./frontend.nix { };
+  vfsMappings = [
+    "config.toml.sample"
+    "schema.sql"
+    "queries.sql"
+    "static/public:/public"
+    "static/email-templates"
+    "${frontend}:/admin"
+    "i18n:/i18n"
+  ];
+  putIntoBinary = mapping: ''
+    ${stuffbin}/bin/stuffbin -a stuff -in $out/bin/listmonk -out $out/bin/listmonk "${mapping}"
+  '';
+in
+buildGoModule rec {
+  pname = "listmonk";
+  version = import ./version.nix;
+
+  vendorSha256 = import ./vendor-hash.nix;
+
+  src = fetchFromGitHub {
+    owner = "knadh";
+    repo = "listmonk";
+    rev = "v${version}";
+    sha256 = import ./source-hash.nix;
+  };
+
+  ldflags = [ "-s" "-w" "-X main.version=${version}" ];
+  dontStrip = false;
```

Comment:
```suggestion
```
The binaries are already stripped, still we should not disable stripping without good reason

Reply:
Yes, it was there because I was debugging issues with stripping, `stuffbin` writes at the end of the binary and strip removes this. So I run stuffbin *after* the stripping.